libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.spec.ts

```diff
@@ -45,51 +47,139 @@ describe(WattNavListModule.name, () => {
+    const view = await render(
+      `
+      <watt-nav-list>
+        <watt-nav-list-item link="/default-page">Default page</watt-nav-list-item>
+        <watt-nav-list-item link="/other-page">Other page</watt-nav-list-item>
+      </watt-nav-list>
+      <router-outlet></router-outlet>
+    `,
+      {
+        declarations: [TestPageComponent],
+        imports: [WattNavListModule, RouterModule],
+        routes: [
+          {
+            path: '',
+            pathMatch: 'full',
+            redirectTo: 'default-page',
+          },
+          {
+            path: 'default-page',
+            component: TestPageComponent,
+          },
+          {
+            path: 'other-page',
+            component: TestPageComponent,
+          },
+        ],
+      }
+    );
 
-    const view = await render(TestAppComponent, {
-      declarations: [TestPageComponent],
-      imports: [WattNavListModule],
-      routes: [
-        {
-          path: '',
-          children: [
-            {
-              path: 'page-1',
-              component: TestPageComponent,
-            },
-            {
-              path: 'page-2',
-              component: TestPageComponent,
-            },
-          ],
-        },
-      ],
+    const activeClass = 'active';
+    const otherPageLink = await screen.findByRole('link', {
+      name: /other page/i,
     });
 
-    const expectedClass = 'active';
+    await view.navigate('/');
 
-    let link = view.getByRole('link', {
-      name: /page 2/i,
-    }) as HTMLAnchorElement;
+    expect(otherPageLink).not.toHaveClass(activeClass);
 
-    expect(Object.values(link.classList)).not.toContain(expectedClass);
+    await view.navigate(otherPageLink);
 
-    await view.navigate(link);
+    expect(otherPageLink).toHaveClass(activeClass);
+  });
 
-    link = view.getByRole('link', {
-      name: /page 2/i,
-    }) as HTMLAnchorElement;
+  it('Supports external links', async () => {
+    // Arrange
+    await render(
+      `
+      <watt-nav-list>
+        <watt-nav-list-item link="https://energinet.dk">
+          Energinet
+        </watt-nav-list-item>
+      </watt-nav-list>
+    `,
+      {
+        imports: [WattNavListModule],
+      }
+    );
+
+    // Act
+
+    // Assert
+    expect(
+      await screen.findByRole('link', {
+        name: /energinet/i,
+      })
+    ).toHaveAttribute('href', 'https://energinet.dk');
+  });
+
+  it('Ensures external links are specified with the expected protocol', async () => {
+    // Arrange
+    await render(
+      `
+      <watt-nav-list>
+        <watt-nav-list-item link="https://energinet.dk">
+          Energinet
+        </watt-nav-list-item>
+      </watt-nav-list>
+    `,
+      {
+        imports: [WattNavListModule],
+      }
+    );
+
+    // Act
+    const link = await screen.findByRole('link', {
+      name: /energinet/i,
+    });
+    const href = link.getAttribute('href');
+
+    // Assert
+    expect(href).toMatch(/^(http:\/\/|https:\/\/)/i);
```

Comment:
suggestion: use a DOM matcher
```suggestion
    const link = await screen.findByRole('link', {
      name: /energinet/i,
    });

    // Assert
    expect(link).toHaveAttribute('href', 'https://energinet.dk');
```

Reply:
issue: You still haven't verified that it works with an external link using the `http://` protocol 🙃